operator/option/config.go

```diff
@@ -41,6 +41,10 @@ const (
 	// compatible with MetalLB's configuration.
 	BGPConfigPath = "bgp-config-path"
 
+	// SkipCRDCreation specifies whether the CustomResourceDefinition will be
+	// disabled for the daemon
```

Comment:
It's not for the daemon which refers to the cilium-agent though, so this should say Operator?

Reply:
agree, updated